Code/Framework/AzCore/Tests/Math/SphereTests.cpp

```diff
@@ -29,18 +29,18 @@ namespace UnitTest
     TEST(MATH_Sphere, TestConstructFromVec3AndRadius)
     {
         AZ::Sphere sphere1(AZ::Vector3(10.f, 10.f, 10.f), 15.f);
-        EXPECT_TRUE(sphere1.GetCenter() == AZ::Vector3(10.f, 10.f, 10.f));
-        EXPECT_TRUE(sphere1.GetRadius() == 15.f);
+        EXPECT_EQ(sphere1.GetCenter(), AZ::Vector3(10.f, 10.f, 10.f));
+        EXPECT_EQ(sphere1.GetRadius(), 15.f);
```

Comment:
update: It might be best here to use an IsClose check (due to blasted floating point precision issues). For the single float value you can use `EXPECT_NEAR`, and for the `Vector3` you can do `EXPECT_THAT(sphere1.GetCenter(), IsClose(AZ::Vector3(....))` (you might need an include for this where the custom matcher lives